sources/geom.json

```diff
@@ -3,8 +3,8 @@
 	"fullname": "geom",
 	"name": "geom",
 	"prefix": "geom",
-    	"key":"id",
+	"key": "id",
```

Comment:
Huh? @thisisaaronland do you know why `geom.json` is in this sources repo?

Did they get generated from any top-level property names automatically? Do they need to be included here?